<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analsys</class>
 <widget class="AnalysisTab" name="analsys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>686</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnalyseSystem</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="list_widget" native="true">
     <layout class="QVBoxLayout" name="analsys_layout" stretch="0,0,0,0">
      <item>
       <widget class="QRadioButton" name="button_0">
        <property name="statusTip">
         <string>Executes showd1d and plots an animated density evolution.</string>
        </property>
        <property name="text">
         <string>Plot 1D density evolution</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="button_1">
        <property name="statusTip">
         <string>Executes showsys -nopes and plots an animated 2D density evolution.</string>
        </property>
        <property name="text">
         <string>Plot 2D density evolution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="button_2">
        <property name="statusTip">
         <string>Executes statepop and plots populations for each state.</string>
        </property>
        <property name="text">
         <string>Plot diabatic state population</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="button_3">
        <property name="statusTip">
         <string>Executes showsys -pes and plots a line/contour plot for the PES along the specified coordinates.</string>
        </property>
        <property name="text">
         <string>Plot potential energy surface</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="den1d_box">
     <property name="title">
      <string>1D density options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="den1d_dof_label">
        <property name="text">
         <string>Degree of freedom</string>
        </property>
        <property name="buddy">
         <cstring>den1d_dof</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="den1d_state">
        <property name="specialValueText">
         <string>Sum over states</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="den1d_dof">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="den1d_state_label">
        <property name="text">
         <string>State</string>
        </property>
        <property name="buddy">
         <cstring>den1d_state</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="den2d_box">
     <property name="title">
      <string>2D density options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="den2d_state_label">
        <property name="text">
         <string>State</string>
        </property>
        <property name="buddy">
         <cstring>den2d_state</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="den2d_state">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="den2d_coord_label">
        <property name="text">
         <string>Coordinate Selection</string>
        </property>
        <property name="buddy">
         <cstring>den2d_coord_box</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QScrollArea" name="den2d_coord_box">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="CoordinateSelector" name="den2d_coord">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>312</width>
           <height>85</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="showpes_box">
     <property name="title">
      <string>PES options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="0" column="1">
       <widget class="QComboBox" name="showpes_type">
        <item>
         <property name="text">
          <string>Diabatic PES</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Adiabatic PES</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="showpes_type_label">
        <property name="text">
         <string>Type</string>
        </property>
        <property name="buddy">
         <cstring>showpes_type</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="showpes_state_label">
        <property name="text">
         <string>State</string>
        </property>
        <property name="buddy">
         <cstring>showpes_state</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="showpes_state">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="showpes_coord_label">
        <property name="text">
         <string>Coordinate selection</string>
        </property>
        <property name="buddy">
         <cstring>showpes_coord_box</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QScrollArea" name="showpes_coord_box">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="CoordinateSelector" name="showpes_coord">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>312</width>
           <height>85</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="analyse">
     <property name="text">
      <string>Analyse</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnalysisTab</class>
   <extends>QWidget</extends>
   <header>analysis_gui.ui.analysis_tab</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CoordinateSelector</class>
   <extends>QWidget</extends>
   <header>analysis_gui.ui.coord_select</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
